src/tests/README.md

```diff
@@ -9,42 +9,42 @@ This is OpenVINO testing framework. OpenVINO test system contains:
   * Unit test folder for a particular module should replicate `SRC` folder layout of the corresponding tested module to
     allow further developers get better understanding which part of software is already covered by unit tests and where
     to add new tests if needed.
-    > **Example**: There are `network_serializer.h` and `network_serializer.cpp` files within the `src` folder of the
-    tested OpenVINO module. Then, a new `network_serializer_test.cpp` file should be created within the root of
+    > **Example**: There are `compiled_model.hpp` and `compiled_model.cpp` files within the `src` folder of the
+    tested OpenVINO module. Then, a new `compiled_model_test.cpp` file should be created within the root of
     the Unit Test folder for this module. This test file should cover all the classes and methods from the original
     files.
 
-    > **Example**: There is the `ie_reshaper.cpp` file within the `src/shape_infer` subfolder of the tested module. In this case,
+    > **Example**: There is the `reshape.cpp` file within the `src/shape_infer` subfolder of the tested module. In this case,
     a new `shape_infer` subfolder should be created within the root of the Unit Test folder for this module. And a new
-    `ie_reshaper_test.cpp` file should be created within this newly created subfolder. This test file should cover all
+    `reshape_test.cpp` file should be created within this newly created subfolder. This test file should cover all
     the classes and methods from the original file.
 
   * Each Unit Test should cover only the target classes and methods. If needed, all external interface components should
     be mocked. There are common mock objects provided within the common Unit Test Utilities to stub the general
     OpenVINO API classes.
-    > **Example**: There are `cnn_network_impl.hpp` and `cnn_network_impl.cpp` files within the `src` folder of the tested
-    module. In this case, a new `cnn_network_impl_test.cpp` file should be created and it should contain tests on
-    `CNNNetworkImpl` class only.
+    > **Example**: There are `iplugin.hpp` and `iplugin.cpp` files within the `src` folder of the tested
+    module. In this case, a new `iplugin_test.cpp` file should be created and it should contain tests on
+    `IPlugin` class only.
 
   * It is not prohibited to have several test files for the same file from the tested module.
   * It is not prohibited to create a separate test file for specific classes or functions (not for the whole file).
 
 * **Functional tests**
   This test type is used to verify public OpenVINO API. There are following types of functional tests:
-  * `inference_engine_tests` are plugin-independent tests. They are used to verify OpenVINO API methods that do not
-    involve any plugin runtime. The examples are: `network_reader`, `network_serializer`, and `precision` tests.
+  * `openvino_tests` are plugin-independent tests. They are used to verify OpenVINO API methods that do not
```

Comment:
Originally, `inference_engine_tests` and `plugin_tests` were mean folders with specific tests. For the moment we have `functional/plugin/`, `functional/.../single_op/` and `functional/.../subgraph/`. In this case we need to actualize the document, but may be done in a separate PR